### PR TITLE
added documentation for doctor command

### PR DESCRIPTION
**Pull request by browningjp**
Added documentation for the `doctor` alias of the `dvc version` command. The related code PR fixes issue #3185

> You may disregard these recommendations if you used the **Edit on GitHub** button from dvc.org to improve a doc in place.

❗ Please read the guidelines in the [Contributing to the Documentation](https://dvc.org/doc/user-guide/contributing/docs) list if you make any substantial changes to the documentation or JS engine.

🐛 Please make sure to mention `Fix #issue` (if applicable) in the description of the PR. This causes GitHub to close it automatically when the PR is merged.

Please choose to [allow us](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork) to edit your branch when creating the PR.

Thank you for the contribution - we'll try to review it as soon as possible. 🙏

### CODE REVIEW

**Comment by browningjp** [comment]
Related code PR here https://github.com/iterative/dvc/pull/4722

File: content/docs/command-reference/doctor.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# doctor
+
+Alias for `dvc version`. Display the DVC version along with environment and
+project information.
```
Comment (jorgeorpinel) [inline content/docs/command-reference/doctor.md:4]:
```suggestion
Alias for `dvc version`

Display the DVC version along with environment and project information.
```

File: content/docs/command-reference/doctor.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+## Synopsis
+
+```usage
+usage: dvc doctor [-h] [-q | -v]
+```
+
+## Description
+
+For full details, see [`version`](/doc/command-reference/version).
```
Comment (jorgeorpinel) [inline content/docs/command-reference/doctor.md:14]:
```suggestion
Refer to `dvc version` for details.
```

**Review by jorgeorpinel — APPROVED** [review]
Thanks @browningjp. Can we also link to `dvc doctor` from the `dvc version`? Note that `` `dvc doctor` `` will be linked automatically, so no need for md links in this case.

**Comment by browningjp** [comment]
Hi @jorgeorpinel 

Had a further thought about this. Since it's just an alias, I wonder if we should just eliminate the doctor page entirely, and instead:

1. Put a "doctor" link in the sidebar, that takes you straight to the "version" page
2. Add a note near the top of the "version" page, mentioning the "doctor" alias

This would eliminate needing to duplicate changes, if (for example) the arguments changed in the future. What do you think?

**Comment by jorgeorpinel** [comment]
> Put a "doctor" link in the sidebar, that takes you straight to the "version" page
> Add a note near the top of the "version" page, mentioning the "doctor" alias

Like Wikipedia? Yeah, that is a better idea actually. I'm not sure the sidebar.json will support 2 entries with to the same doc but please try and we'll check on the review app I just created for this: https://dvc-landing-master-q31rvlxpu5a.herokuapp.com/

I'm still confused about this alias thing BTW, though, but that's a discussion that will hopefully take place in the core repo.

**Comment by efiop** [comment]
@browningjp Thank you so much for the PR! We'll take it from here :slightly_smiling_face:

**Comment by shcheklein** [comment]
@jorgeorpinel @browningjp let's not complicate this. I don't quite understand the importance of the change at all (the way I see it - in the original ticket there was an ongoing discussion that is not resolved yet). Since we merged this on the DVC core end, let's merge it here as-is, we spent already too much time on this already to my mind.

**Comment by jorgeorpinel** [comment]
Yup @shcheklein this was already approved and waiting for the core PR merge only. I liked Jonny's idea to simplify the change (avoid introducing a new doc) but OK, not a big deal I suppose. We just have to remember changing both pages if `version` changes.

**Pull request merged**